### ISSUE TITLE
Pokedex pokebattler import disables L5 eggs but raids from webhook don't take this into account

### ISSUE THREAD

**Issue by Artanicus** [florianbecker/PokemonRaidBot#123]
The Pokedex pokebattler import disables L5 eggs if there's only one L5 raidboss, which makes sense. 
However, webhooks will still push in L5 eggs which are then classified as unknown, and certain rules apply unevenly. For example if voting is disabled, it still shows up for unknown since we don't know the level. Oops.

Solution:
The raid from webhook functionality needs to also detect the scenario where only one L5 boss exists and re-route the raid into that L5 boss.

Alternative solution:
If the webhook is enabled in the config, don't disable the L5 eggs. Easier to implement but inconsistent results.

**Comment by Artanicus** [comment]
I've got a sort of a fix for this: https://github.com/Artanicus/PokemonRaidBot/commit/5b78adef06b3f75495b4217274738b410891dc53
TL;DR; add a new option to control raid egg disabling, defaults to old behavior but can be turned off for anyone using webhook ingest that still wants to see the eggs.

It's dependent on config_v2 though, so won't create a PR until that one is merged.

**Comment by Artanicus** [comment]
PR for fixing this is in #125
